### PR TITLE
Fixed logic for max event. mobile to online now supported.

### PR DESCRIPTION
**Pull request by nick110303**

### CODE REVIEW

File: src/events/maxisonline.py [hunk 1]
```diff
@@ -114,8 +114,9 @@ async def on_presence_update(self, before: discord.Member, after: discord.Member
             # Get Max's info only
             if before.id == self.max_member.id:
                 if before.guild.id == 730196305124655176:
-                    if (before.status != discord.Status.online and after.status == discord.Status.online) \
+                    if (before.is_on_mobile() and after.status == discord.Status.online) \
```
Comment (jgionti) [inline src/events/maxisonline.py:117]: Why not `before.is_on_mobile() != after.is_on_mobile()`? It's possible to be mobile and online so this doesn't catch all cases.

**Review by jgionti — CHANGES_REQUESTED** [review]

File: src/events/maxisonline.py (hunk 1 above)
Comment (nick110303) [reply src/events/maxisonline.py]: ok

File: src/events/maxisonline.py [hunk 2]
```diff
@@ -114,8 +114,9 @@ async def on_presence_update(self, before: discord.Member, after: discord.Member
             # Get Max's info only
             if before.id == self.max_member.id:
                 if before.guild.id == 730196305124655176:
-                    if (before.status != discord.Status.online and after.status == discord.Status.online) \
+                    if (before.is_on_mobile() and after.is_on_mobile()) \
```
Comment (jgionti) [inline src/events/maxisonline.py:117]: Not what I said :)

If the user stays on their phone that doesn't qualify a post does it?

**Review by jgionti — CHANGES_REQUESTED** [review]

File: src/events/maxisonline.py (hunk 2 above)
Comment (nick110303) [reply src/events/maxisonline.py:117]: mistype
very loud music here

**Review by jgionti — APPROVED** [review]
Looks good. Please **squash and merge** (see the merge options) then make sure that everything works in prod.

**Comment by nick110303** [comment]
all done

**Pull request merged**